src/classes/CW_AudienceGroupCompControlerTest.cls

```diff
@@ -0,0 +1,270 @@
+/**
+* File Name   :    CW_AudienceGroupCompControlerTest.cls
+* Description :   
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 03/04/2014 Cristina P		Initial
+*/
+@isTest
+public with sharing class CW_AudienceGroupCompControlerTest {
+	
+	public static CW_Campaign_Navigation_Settings__c 	cns1;
+	public static CW_Campaign_Navigation_Settings__c 	cns2;
+	
+
+	
+	public static void initCustomSettings()
+	{
+		cns1 = TestDataUtil.createCampaignNavigationSettings(Page.CW_Campaign_Information,1,false);
+    	cns2 = TestDataUtil.createCampaignNavigationSettings(Page.CW_Audience,2,false);
+    	insert new List<CW_Campaign_Navigation_Settings__c>{cns1,cns2};
+    	
+    	CW_CampaignListSettingHelper.getIntegerValue(ConstantUtil.AUDIENCE_GROUP_PAGE_SIZE,5);
+	}
+	
+	static testMethod void standardSetControllerTest() 
```

Comment:
Style issue: test methods should be annotated with @IsTest for better readibility